redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -217,6 +227,32 @@ public static function updateMedia(string $filename, array $data): array
             }
         }
 
+        // Set dynamic fields from $data, such as 'med_description'
+        foreach ($data as $key => $value) {
```

Comment:
Mich stört hier bisschen, dass die Felder am metainfo-Addon vorbei einfach direkt in die DB gespeichert werden.
Also wenn das Metainfo-Addon z.b. mal Validierungen einführen würde, würden die hier übergangen werden.

Andererseits ist es natürlich auch pragmatisch/simpel, es hier einfach so direkt zu machen.